feat: added translations specs (#124)
algolia/instantsearch-specs: content/widgets/smart-sort.md

    description: the informative text
  - name: .ais-RelevantSort-button
    description: the button text
translations:
  - name: informationText
    default: '({ isRelevantSorted, isVirtualReplica }) => ""'

>     description: An information text that is displayed above the button.

Reviewer: ```suggestion
    description: Information text displayed above the button.
```

  - name: seeButtonText
    default: '({ isRelevantSorted, isVirtualReplica }) => isRelevantSorted ? "See all results" : "See relevant results"'

>     description: The text that will be displayed on the "See *** results" button.

Reviewer: ```suggestion
    description: The text of the "See _n_ results" button.
```
